### DISCUSSION TITLE
package.json main field points to non-existent file

### DISCUSSION THREAD

**Discussion by longlho** [mercadopago/sdk-js#171] [Q&A] [answered]
Hi guys, in your `package.json`, the `main` field points to `index.js` which doesn't exist in your dist. Only `dist/index.js` exists. This seems to fail our jest tests. Can this be fixed?

**Comment by AleFossati** [comment, marked as answer]
Hello, thanks for your [pull request](https://github.com/mercadopago/sdk-js/pull/172).
